content/intro-to-storybook/react-native/en/simple-component.md

```diff
@@ -75,167 +125,108 @@ There are two basic levels of organization in Storybook: the component and its c
-
-`action()` allows us to create a callback that appears in the **actions** panel of the Storybook UI when clicked. So when we build a pin button, we’ll be able to determine in the test UI if a button click is successful.
+To tell Storybook about the component we are documenting, we create a `default` export that contains:
 
-As we need to pass the same set of actions to all permutations of our component, it is convenient to bundle them up into a single `actions` variable and use React's `{...actions}` props expansion to pass them all at once. `<Task {...actions}>` is equivalent to `<Task onPinTask={actions.onPinTask} onArchiveTask={actions.onArchiveTask}>`.
+- `component` -- the component itself
+- `title` -- how to refer to the component in the sidebar of the Storybook app
 
-Another nice thing about bundling the `actions` that a component needs is that you can `export` them and use them in stories for components that reuse this component, as we'll see later.
+Some optional values we also added were
 
-To define our stories, we call `add()` once for each of our test states to generate a story. The action story is a function that returns a rendered element (i.e. a component class with a set of props) in a given state---exactly like a [Stateless Functional Component](https://reactjs.org/docs/components-and-props.html).
+- decorators -- allows you to wrap a component and in this case apply some styling
+- argTypes -- allows us to specify the types of our args, here we use it to define actions which will log whenever that interaction takes place
 
-When creating a story we use a base task (`task`) to build out the shape of the task the component expects. This is typically modelled from what the true data looks like. Again, `export`-ing this shape will enable us to reuse it in later stories, as we'll see.
+To define our stories we export an object with an `args` property. Arguments or [`args`](https://storybook.js.org/docs/react/writing-stories/args) for short, allow us to live-edit our components with the controls addon without restarting Storybook. Once an [`args`](https://storybook.js.org/docs/react/writing-stories/args) value changes, so does the component.
 
-<div class="aside">
-<a href="https://storybook.js.org/docs/react/essentials/actions"><b>Actions</b></a> help you verify interactions when building UI components in isolation. Oftentimes you won't have access to the functions and state you have in context of the app. Use <code>action()</code> to stub them in.
-</div>
+When creating a story, we use a base `task` arg to build out the shape of the task the component expects. Typically modeled from what the actual data looks like. Again, `export`-ing this shape will enable us to reuse it in later stories, as we'll see.
 
-## Config
+Now that we've set up the basics lets re-run `yarn storybook` and see our changes.
```

Comment:
@dannyhw, leave the config section in and reframe it, if possible to:
- Showcase how the React Native version detects the stories
- Refrain from mentioning deleting the default stories (leaving them in serves as guidelines for folks to see other examples).